tensorflow/contrib/cmake/CMakeLists.txt

```diff
@@ -219,64 +230,114 @@ if(UNIX)
 endif()
 
 if (tensorflow_ENABLE_GPU)
+  if (NOT WIN32)
+    # Default install paths for cuda libraries in Linux
+    list(APPEND CMAKE_LIBRARY_PATH /usr/local/cuda/lib64)
+    list(APPEND CMAKE_LIBRARY_PATH /usr/local/cuda/lib64/stubs)
```

Comment:
Is there a way to get this information from `find_package(CUDA 8.0 REQUIRED)` or should this be an option with a default value?

Reply:
In my case, find_package(CUDA 8.0 REQUIRED) had failed if CMAKE_LIBRARY_PATH did not include /usr/local/cuda/lib64. This could be caused by incomplete packaging code of my own CUDA RPM packaging code; however, I'm not sure why yet.

Reply:
Perhaps we could move these inline paths to options? (It seems redundant with `tensorflow_PATH_STATIC_LIB`, for example.)

Reply:
Adding options for these (additional ```CMAKE_LIBRARY_PATH```) looks better.
However, they are not redundant with ```tensorflow_PATH_STATIC_LIB``` because they are not meant to be static libraries (.a); they are for shared libraries (.so).

Reply:
I've added the option:
```
option(tensorflow_CUDA_LIBRARY_PATH "Designate the default CUDA library paths" /usr/local/cuda/lib64)
```